application.go

```diff
@@ -180,6 +181,13 @@ func (r *Application) CPU(cpu float64) *Application {
 	return r
 }
 
+// GPU set the amount of GPU per instance which is assigned to the application
+//		gpu:	the GPU (check MESOS docs) per instance
+func (r *Application) GPU(gpu float64) *Application {
```

Comment:
Can we call this `GPUs` for consistency reasons?

(I know we have a `CPU()` method, but I think it should be renamed to `CPUs()` at some point too.)